diff -Nrup task-1.7.1-1/CYGWIN-PATCHES/setup.hint task-1.7.1-1.cygwin/CYGWIN-PATCHES/setup.hint
--- task-1.7.1-1/CYGWIN-PATCHES/setup.hint	1970-01-01 01:00:00.000000000 +0100
+++ task-1.7.1-1.cygwin/CYGWIN-PATCHES/setup.hint	2009-06-17 15:47:56.304750000 +0200
@@ -0,0 +1,10 @@
+# setup.hint for task 1.7.1-1
+category: Utils
+requires: libncurses9 cygwin 
+sdesc: A command-line to do list manager
+ldesc: "Task is a command-line to do list manager.
+It has support for GTD functionality and includes
+the following features: tags, colorful tabular output,
+reports and graphs, lots of manipulation commands,
+low-level API, abbreviations for all commands and
+options, multiuser file locking, recurring tasks."
diff -Nrup task-1.7.1-1/CYGWIN-PATCHES/task-1.7.1-1.README task-1.7.1-1.cygwin/CYGWIN-PATCHES/task-1.7.1-1.README
--- task-1.7.1-1/CYGWIN-PATCHES/task-1.7.1-1.README	1970-01-01 01:00:00.000000000 +0100
+++ task-1.7.1-1.cygwin/CYGWIN-PATCHES/task-1.7.1-1.README	2009-06-17 15:47:44.320375000 +0200
@@ -0,0 +1,45 @@
+task 1.7.1-1
+------------------------------------------
+
+A command-line to do list manager that can be used
+with the gtd system.
+
+Runtime requirements:
+  cygwin-1.5.25 or newer
+  libncurses
+
+Build requirements:
+  cygwin-1.5.25 or newer
+  gcc-3.4.4-1 or newer
+  libncurses-devel
+
+Canonical homepage:
+  http://taskwarrior.org
+
+Canonical download:
+  http://taskwarrior.org/download
+
+License:
+  GPLv2+
+
+Language:
+  C++
+
+------------------------------------
+
+Build instructions:
+  unpack task-1.7.1-1-src.tar.bz2
+    if you use setup to install this src package,
+    it will be unpacked under /usr/src automatically
+  cd /usr/src/task-1.7.1-1
+  ./configure --prefix=/usr
+  make
+  make install
+
+------------------------------------
+
+task was packaged for cygwin by
+Federico Hernandez <ultrafredde@gmail.com> and
+is licensed under the GPL
+
+---- task-1.7.1-1 -- 2009-06-17 ----
